.claude/briefs/BRIEF_2025-10-31.md

```
@@ -0,0 +1,114 @@
+# Codebase Brief: horizons (feature-library-predictions branch)
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)